docs/advanced-features/workflows.md

```diff
@@ -623,6 +629,31 @@ Requires a README
 }
 ```
 
+##### `s3://bucket/must-have-foo-bar-meta.json`
```

Comment:
@sir-sigurd please, check if the docs, and example in particular, are correct

Reply:
I guess I need to drop my example in https://github.com/quiltdata/quilt/pull/3286/files, since your doc seems more complete.